### DISCUSSION TITLE
Welcome to simple-microservice-nodejs Discussions!

### DISCUSSION THREAD

**Discussion by DzunN86** [DzunN86/simple-microservice-nodejs#4] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asuna199** [comment, marked as answer]
What database do you use ?

**Reply by DzunN86** [reply]
MySQL

**Comment by DzunN86** [comment]
This is a simple microservices. I am using node js and laravel for this stuff

**Reply by asuna199** [reply]
express and laravel ?

**Reply by DzunN86** [reply]
exactly
